### ISSUE TITLE
Fehler bei der Ausführung (step2)

### ISSUE THREAD

**Issue by KutSaleh** [medizininformatik-initiative/Projectathon6-miracum1#9]
Skript wirft diese Fehlermeldung bei der Ausführung aus: 

Aggregating results
Fitting final model on full training set
[1] "Fitting models for Ischemic_count for daily resolution"
[1] "split to train and test set"
[1] "Poisson"
**Error in match.arg(type) : 
  'arg' sollte eines von '“link”, “response”, “terms”' sein**

**Comment by mematt** [comment]
Thank you for raising this issue. 
It is caused by copy-paste mistake incorporating `predict(..., type = "raw")` instead of `predict(..., type = "response")` in all predict.glm() calls.  

We are fixing it throughout the script.

**Comment by NandhiniS08** [comment]
Issue fixed and ran successfully in current form
